src/trivial_demonstrations/greatest_of_three_numbers

```diff
@@ -0,0 +1,11 @@
+fn main() {
+    let x = 10;
+    let y = 25;
+    let z=30;
+    if x>y && x>z{
+        println!("{}",x);}
+    if y>z && y>x{
+        println!("{}",y);}
+    else{
```

Comment:
Conditions seem to print wrong output.
Please check the logic again and run the program either in your own system or on any online compiler see the program is yielding recieved output before pushing to repository